createdisk.sh

```diff
@@ -55,8 +55,9 @@ if [ ${BUNDLE_TYPE} != "microshift" ]; then
     ${SSH} core@${VM_IP} -- sudo systemctl stop kubelet
 fi
 
-# Enable the podman.socket service for API V2
+# Enable the system and user level  podman.socket service for API V2
 ${SSH} core@${VM_IP} -- sudo systemctl enable podman.socket
+${SSH} core@${VM_IP} -- sudo systemctl --user enable podman.socket
```

Comment:
do we need `sudo` here since using `--user`?

Reply:
updated.